drivers/SmartThings/matter-lock/src/new-matter-lock/init.lua

```diff
@@ -1660,6 +1689,7 @@ local new_matter_lock_handler = {
     [capabilities.lock.ID] = {
       [capabilities.lock.commands.lock.NAME] = handle_lock,
       [capabilities.lock.commands.unlock.NAME] = handle_unlock,
+      [capabilities.lock.commands.unlatch.NAME] = handle_unlatch
```

Comment:
I believe `supportedLockCommands` needs to be set appropriately as well, either including `unlatch` or not, depending on whether the device supports the UBOLT feature